.clang-format

```diff
@@ -0,0 +1,185 @@
+# This file represents the coding style enforced by Open MPI. This file
+# is based on the long-held, but not enforced, guidelines from the
+# beginning of the project. We will be requiring that all code going
+# forward uses this style. To check your code before attempting to open
+# a PR install clang-format and run your commits through clang-format.
+#
+# To install clang-format:
+#
+# macOS:
+#   Homebrew: brew install clang-format
+#   Mac Ports: port install clang-format
```

Comment:
I tried this MacPorts cmd and got a "no such package" error

Reply:
Aha! It is apparently installed along with clang now, so no separate package required

Reply:
Ok. Will fix. At one point it was separate but being included is great.